modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -21,16 +21,17 @@ export interface GenerateMpcWalletOptions {
   originalPasscodeEncryptionCode?: string;
   enterprise?: string;
   walletVersion?: number;
-  backupProvider?: string;
+  backupProvider?: BackupProvider;
 }
-
+export const backupProviders = ['BitGoTrustAsKrs'] as const;
+export type BackupProvider = typeof backupProviders[number];
```

Comment:
NIT. This could be a bit simpler
```suggestion
export type BackupProvider = 'BitGoTrustAsKrs'
```
Adding more type would simply be 
`export type BackupProvider = 'BitGoTrustAsKrs' | 'next type'`